x/tx/signing/get_signers.go

```diff
@@ -175,3 +187,37 @@ func (c *GetSignersContext) GetSigners(msg proto.Message) ([]string, error) {
 
 	return f(msg), nil
 }
+
+// GetSignersForAny returns the signers for a given proto message wrapped inside
+// an Any. It can be used to retrieve the signers for a message which does not
+// have a concrete protoreflect type in the current binary because it uses dynamicpb
+// under the hood. This is useful for usage in dynamic clients (like autocli) or
+// for working with legacy gogo protobuf messages.
+func (c *GetSignersContext) GetSignersForAny(any *anypb.Any) ([]string, error) {
+	url := any.TypeUrl
+	typ, err := c.protoTypes.FindMessageByURL(url)
+	if err != nil {
+		if err == protoregistry.NotFound {
```

Comment:
For now this same code will be in 2 places, here and in textual. If we'll need it in a third place, I think it's worth thiking about where to put it for DRY.

Reply:
It's also used in autocli. Where should that util live? cosmos-proto?

Reply:
Yeah I think putting it in the same package as our own Any utils is fine.

Reply:
I'm actually wondering if the any conversion should be dealt with by the caller. If we have a util function that will be easier and it would simplify what is needed here

Reply:
Here's the util: https://github.com/cosmos/cosmos-proto/pull/101.

Yeah, if the caller uses that util, then it's 4 lines to get the signers. I don't think it's worth having this additional method TBH.